app/jobs/create_institution_records_job.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'net/http'
+
+class CreateInstitutionRecordsJob < ApplicationJob
+  queue_as :default
+
+  def perform(institution_type_id)
+    i = 1
+    loop do
+      rspo_api_uri = URI("http://194.54.26.132/api/placowki/?page=#{i}&typ_podmiotu_id=#{institution_type_id}")
+      request = Net::HTTP::Get.new(rspo_api_uri)
+      request['accept'] = 'application/json'
+      response = Net::HTTP.start(rspo_api_uri.hostname, rspo_api_uri.port) do |http|
+        http.request(request)
+      end
+
+      raw_institutions = JSON.parse(response.body)
+
+      # the records are sent in pages of 100
+      break if raw_institutions == []
+
+      i += 1
+
+      batch_insert_institutions = []
+      raw_institutions.each do |raw_institution|
+        institution = {}
+        institution['institution_type_id'] = raw_institution['typ']['id']
+        institution['name'] = raw_institution['nazwa']
+        institution['rspo_institution_id'] = raw_institution['numerRspo']
+        institution['public'] = raw_institution['statusPublicznoPrawny']['nazwa'] == 'publiczna'
+        institution['latitude'] = raw_institution['geolokalizacja']['latitude'].to_f
+        institution['longitude'] = raw_institution['geolokalizacja']['longitude'].to_f
+        institution['website'] = raw_institution['stronaInternetowa']
+        institution['email'] = raw_institution['email']
+        # rubocop:disable Layout/LineLength
+        institution['address'] = "#{raw_institution['adresDoKorespondecjiMiejscowosc']} #{raw_institution['adresDoKorespondecjiUlica']} #{raw_institution['adresDoKorespondecjiNumerBudynku']}/#{raw_institution['adresDoKorespondecjiNumerLokalu']} #{raw_institution['adresDoKorespondecjiKodPocztowy']}"
```

Comment:
If you're expecting a hash always to include a specific value, it's a good idea to use [`Hash#fetch`](https://apidock.com/ruby/Hash/fetch)
Now, if you make a mistake in the key's name, it will insert an empty value in the DB, which can be easy to miss. `fetch` will throw an exception, which is impossible to ignore. It will also immediately notify you if the API's response changes.